src/main/java/seedu/address/logic/commands/DeleteProjectCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.employee.Employee;
+import seedu.address.model.project.Project;
+
+/**
+ * Deletes a project using its identified index from the TaskHub.
+ */
+public class DeleteProjectCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteP";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the project identified by the index number used in the displayed project list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_PROJECT_SUCCESS = "Deleted Project: %1$s";
+
+    private final Index targetIndex;
+    public DeleteProjectCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Project> lastShownList = model.getFilteredProjectList();
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX);
+        }
+        Project projectToDelete = lastShownList.get(targetIndex.getZeroBased());
+        model.deleteProject(projectToDelete);
+
+        projectToDelete.getEmployees().asUnmodifiableObservableList().stream().forEach(employee -> {
+            model.setEmployee(employee, new Employee(employee.getName(), employee.getPhone(),
+                    employee.getEmail(), employee.getAddress(), new Project(""), employee.getTags()));
+        });
```

Comment:
Sorry! I don't understand what this part is doing. Can you please help clarify?

Reply:
For each employee in the project list, the project attribute is set to an empty string (the employee is not associated with the project). This may cause confusion due to the "multiple projects per person" setting, and this part will be updated when the employee takes a list of projects instead.